Sources/SwiftFusion/Core/FixedShapeTensor.swift

```diff
@@ -0,0 +1,77 @@
+import TensorFlow
```

Comment:
Maybe provide some docs and rationale for why this module exists.

Reply:
Ok, I'll do this in a follow up PR!